### PR TITLE
Feature/miguel

### PR DESCRIPTION
**Pull request by MegaBrute**
Added Program and Encounter class

### CODE REVIEW

**Review by wtvamp — APPROVED** [review]
Good stuff

**Review by GrpFruit — APPROVED** [review]

**Review by jmorales8 — APPROVED** [review]

**Review by ajcruzmart — COMMENTED** [review]
Nice Miguel!

File: Program.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        { 
+        case "1":
+            Console.WriteLine(hero1.characterName + " stumbled against a zombie.");
+            break;
+        case "2":
+            Console.WriteLine(hero1.characterName + " got wrecked by numerous forces.");
+            System.Environment.Exit(0x0);
+            break;
+        }
+        Encounter fightzombie = new Encounter(hero1, new Monster("Zombie", 100,"rauggghh", "grey", new Weapon( 3,3, "Bite","physical")));
```
Comment (ajcruzmart) [inline Program.java:134]: Cool

**Review by ajcruzmart — APPROVED** [review]
Nice!

File: Program.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
+public class Program {
+    
+}
+static void Main(string[] args)
+{
+    // WJT: Leave comments like this at points where you get stuck and need me to take a look
+    //Begining of game - scene 1 
+    
+    Console.WriteLine ("========================================");
```
Comment (BreadToastt) [inline Program.java:9]: ohh spooky

**Pull request merged**